src/Dev/SapphireTest.php

```diff
@@ -395,6 +396,10 @@ protected function currentTestDisablesDatabase()
      */
     public static function setUpBeforeClass(): void
     {
+        // Disallow the use of DB replicas in tests
+        // This prevents replicas from being used if there are environment variables set to use them
+        DB::setCanUseReplicas(false);
```

Comment:
```suggestion
        DB::setMustUsePrimary();
```
Surely we want to _always_ use the primary database for unit tests?

Reply:
Updated, I've removed all of the setCanUseReplicas() related stuff